crates/formality-prove/src/prove.rs

```diff
@@ -53,7 +54,20 @@ pub fn prove(
 
     assert!(env.encloses(term_in));
 
-    let result_set = prove_wc_list(decls, &env, assumptions, goal);
+    struct ProveFailureLabel(String);
```

Comment:
Question: is it even worth including the "decls" here? I'm inclined to think "not really".

Reply:
I think it's nice to see it at least once in proof trace on failure.